tests/Start-DbaAgentJob.Tests.ps1

```diff
@@ -15,12 +15,17 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
 
 Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
     It "returns a CurrentRunStatus of not Idle and supports pipe" {
-        $null = Get-DbaAgentJob -SqlInstance $script:instance2 -Job 'DatabaseBackup - SYSTEM_DATABASES - FULL' | Start-DbaAgentJob
+        $results = Get-DbaAgentJob -SqlInstance $script:instance2 -Job 'DatabaseBackup - SYSTEM_DATABASES - FULL' | Start-DbaAgentJob
```

Comment:
We can't test starting a job that won't exist on the instance in Appveyor. A `BeforeAll` and `AfterAll` needs to be added to create the specified job.

Reply:
Makes sense. I've used this as an example
https://github.com/sqlcollaborative/dbatools/commit/c7e380545a8e73f7ef7bfa1ea98d99fa51f24c56

Will fix it by adding steps to create the job.